astropy/io/fits/tests/test_connect.py

```diff
@@ -24,8 +24,8 @@
 from astropy.table import Column, MaskedColumn, QTable, Table
 from astropy.table.table_helpers import simple_table
 from astropy.time import Time
+from astropy.units import UnitScaleError
```

Comment:
The three astropy.units imports could in principle be combined:
```
from astropy.units.format.fits import UnitScaleError, allclose as quantity_allclose, QuantityInfo
```
Maybe at least do the `QuantityInfo`? We really shouldn't give the bad example of importing from down the subpackages in our own test files...

Reply:
The purpose of the changes here was to reveal the bug. Importing `QuantityInfo` is not related to the bug in question and changing how it's done is beyond the scope here.

If you prefer
```python
from astropy.units.format.fits import UnitScaleError, allclose as quantity_allclose
```
over
```python
from astropy.units import UnitScaleError
from astropy.units import allclose as quantity_allclose
```
then you should open a separate pull request to turn on the [`ruff.lint.isort.combine-as-imports`](https://docs.astral.sh/ruff/settings/#lint_isort_combine-as-imports) setting.